tests/inputFields.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { test, expect } from '@playwright/test';
+
+
+test.describe('Input fields', () => {  
+test.beforeEach( async({page}) => {
+  await page.goto('/')
+  await page.getByRole('link', { name: 'Pet Types' }).click();
+  await page.getByRole('heading', { name: 'Pet Types' }).click()
```

Comment:
Instead of clicking on Pet Types header, you need to make an assertion, that this text is displayed in the header of the text